src/sql/inspect.py

```diff
@@ -44,6 +44,37 @@ def __init__(self, schema=None, conn=None) -> None:
         self._table_txt = self._table.get_string()
 
 
+def _add_missing_keys(keys, mapping):
+    """
+    Return a copy of `mapping` with all the missing `keys`, setting the
+    value as an empty string
+    """
+    return {key: mapping.get(key, "") for key in keys}
+
+
+# we're assuming there's one row that contains all keys, I tested this and worked fine
+# my initial implementation just took all keys that appeared in "rows" but then order
+# isn't preserved, which is important for user experience
+def _get_row_with_most_keys(rows):
+    """
+    Get the row with the maximum length from the nested lists in `rows`
+    """
+    if not rows:
+        return list()
```

Comment:
I am assuming we will be receiving a list with sub lists inside in row attribute. Won't it be better is we can throw a valueError over here, as function is only looking for max elements?